application/src/main/java/org/togetherjava/tjbot/commands/utils/MessageUtils.java

```diff
@@ -56,4 +59,25 @@ public static String escapeMarkdown(String text) {
         return afterEscape.replace("\\```", "\\`\\`\\`");
     }
 
+    /**
+     * Converts a slash command text to a mentioned slash command, which you can directly click on
+     * in Discord.
+     *
+     * @param guild the {@link Guild} that contains the command
+     * @param command the command's name
+     * @return Formatted string for the mentioned slash command
+     * @throws IllegalArgumentException when the command name doesn't match with the guild's
+     *         commands
+     */
+    public static RestAction<String> mentionSlashCommand(Guild guild, String command) {
+        return guild.retrieveCommands().map(commands -> {
+            Command foundCommand = commands.stream()
+                .filter(c -> c.getName().equalsIgnoreCase(command))
+                .findFirst()
```

Comment:
use `findAny()`. its potentially faster. and u dont care which it picks, since u dont know the order either.